spec/grammarkdown.grammar

```diff
@@ -44,26 +51,77 @@ Keyword :: one of
 	`of`
 	`or`
 	`but`
-	`not`	
+	`not`
 
 DecimalDigit :: one of
 	`0` `1` `2` `3` `4` `5` `6` `7` `8` `9`
 
+HexDigits ::
+	HexDigit
+	HexDigits HexDigit
+
 HexDigit :: one of
 	`0` `1` `2` `3` `4` `5` `6` `7` `8` `9` `a` `b` `c` `d` `e` `f` `A` `B` `C` `D` `E` `F`
 
+NonZeroHexDigit ::
+	HexDigit but not `0`
+
 UnicodeCharacterLiteral ::
 	`<` UnicodeCharacterLiteralChars `>`

```

Comment:
This is much more general than the "ASCII identifier name" mentioned in the [README](https://github.com/rbuckton/grammarkdown/tree/main#character-literals-and-ranges), and probably _too_ general given the inclusion of |LineContinuation|. Perhaps strike a middle ground by requiring ASCII for `<U+…>` in accordance with Unicode conventions but allowing arbitrary identifiers in other `<…>` instances?
```suggestion
PrintableASCIIChars ::
	PrintableASCIIChar
	PrintableASCIIChars PrintableASCIIChar

PrintableASCIIChar ::
	U+0020 through U+007F

UnicodeCharacterLiteral ::
	`<` IdentifierName `>`
	`<` `U+` UnicodeCharacterLiteralHexDigits U+0020 PrintableASCIIChars `>`
```

Reply:
I wouldn't worry too much about that file, I need to completely rewrite it